lib/index.js

```diff
@@ -27,6 +27,17 @@ class PWMetrics {
     this.sheets = opts.sheets;
     this.expectations = opts.expectations || false;
     this.tryLighthouseCounter = 0;
+    this.desiredMetrics = [];
+    if (this.flags.desiredMetrics) {
+      this.desiredMetrics = this.flags.desiredMetrics.filter(metric => {
```

Comment:
What you get from input may be different from what pwmetrics can give you. This code validates that. You tell it to run metric XYZ, the desiredMetric flag has content but it is not valid-> remove from array.